server.go

```diff
@@ -883,21 +897,7 @@ func (s *Server) handleStream(t transport.ServerTransport, stream *transport.Str
 	method := sm[pos+1:]
 	srv, ok := s.m[service]
 	if !ok {
-		if trInfo != nil {
-			trInfo.tr.LazyLog(&fmtStringer{"Unknown service %v", []interface{}{service}}, true)
```

Comment:
I know, I extracted it into a single function with an error listing the full method name (service + method). The code otherwise was completely duplicate.

Would you prefer me to return it to the previous state?